client/app/components/ActivityLogComp/index.js

```diff
@@ -21,7 +28,8 @@ class ActivityLogComp extends Component{
             status: activity.Status,
             projectName: activity.Type,
```

Comment:
Create a single object named activity in state and take all required fields of activity as the key of that object